source/_statement.html.md

```diff
@@ -23,36 +30,48 @@ We’re always looking to improve the accessibility of this service. If you find
 The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, contact the [Equality Advisory and Support Service (EASS)](https://www.equalityadvisoryservice.com/).
 
 ## Technical information about this service’s accessibility
-We're committed to making this service accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+The CDIO is committed to making this service accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
```

Comment:
Why not "We're"? 

If we really don't want to say that, then I'd recommend we go with Cabinet Office.

Reply:
Left as is due to the above... again, we can revisit.